test/gateway/gateway.createorder.test.js

```diff
@@ -36,9 +36,9 @@ describe("Gateway create order", function () {
 		this.mockUSDT = mockUSDT;
 		this.gateway = gateway;
 
-		this.mintAmount = ethers.utils.parseEther("27027000");
-		this.orderAmount = ethers.utils.parseEther("27027000");
-		this.protocolFee = ethers.utils.parseEther("27000");
+		this.mintAmount = ethers.utils.parseEther("27000000");
+		this.orderAmount = ethers.utils.parseEther("27000000");
+		this.protocolFee = 0;
```

Comment:
protocolFee should be 27000 as it is not 0

the only difference is the provider amount on settlement will now be 26973000

Reply:
if you check, you will realized that I did that also,